documentation/en/default-curio-config.toml

```diff
@@ -319,16 +319,24 @@
 
 
 [Ingest]
+  # Maximum number of sectors that can be queued waiting for deals to start processing.
+  # 0 = unlimited
+  # Note: This mechanism will delay taking deal data from markets, providing backpressure to the market subsystem.
+  # The DealSector queue includes deals which are ready to enter the sealing pipeline but are not yet part of it -
+  # size of this queue will also impact the maximum number of ParkPiece tasks which can run concurrently.
+  # DealSector queue is the first queue in the sealing pipeline, meaning that it should be used as the primary backpressure mechanism.
+  #
+  # type: int
+  #MaxQueueDealSector = 8
+
   # Maximum number of sectors that can be queued waiting for SDR to start processing.
   # 0 = unlimited
   # Note: This mechanism will delay taking deal data from markets, providing backpressure to the market subsystem.
-  # The SDR queue includes deals which are in the process of entering the sealing pipeline - size of this queue
-  # will also impact the maximum number of ParkPiece tasks which can run concurrently.
-  # 
-  # SDR queue is the first queue in the sealing pipeline, meaning that it should be used as the primary backpressure mechanism.
+  # The SDR queue includes deals which are in the process of entering the sealing pipeline. In case of the trees tasks it is
+  # possible that this queue grows more than this limit, the backpressure is only applied to sectors entering the pipeline.
   #
   # type: int
-  #MaxQueueSDR = 8
+  #MaxQueueSDR = 0
```

Comment:
Should we maybe redefine this in terms of GiBs?

Reply:
I think sector makes a lot more sense. Sizes would cause problem if we have mixed size miner IDs on same cluster. It will make estimates vary a lot.